nesta/packages/unsdg_nlp_utils/preprocess_spacy.py

```diff
@@ -0,0 +1,44 @@
+import spacy
+nlp = spacy.load('en', disable=['ner'])
+
+def convert(text):
+
+    return str(text).replace('“',"").replace('”',"").replace('‘',"'").replace('’',"'")
+
+def spacy_nlp_vocab_update(stop_word_list): #put in main function or leave it here?
+    for word in stop_word_list:
+    l = word.lower()
+    u = word.upper()
+    t = word.title()
+    nlp.vocab[l].is_stop = True
+    nlp.vocab[u].is_stop = True
+    nlp.vocab[t].is_stop = True
+
+def word_tokenise(text):

```

Comment:
It might be useful to give this function some arguments that allow you to choose whether to filter stop words, what the shortest token length should be etc.